### PR TITLE
Adds Chemist Role

### PR DESCRIPTION
**Pull request by Elijahrane**
## About the PR 
Adds the chemist role and everything necessary for it. The chemistry labs on saltern and packedstation have been updated with new airlocks and spawners to accommodate them.
**Screenshots**

unknown

**Changelog**

:cl: Rane
- add: The chemist has arrived on station! Pharmacology will be left to the professionals now.

### CODE REVIEW

**Comment by moonheart08** [comment]
This'll need tied into #5422 should that be merged first, as positions and spawnPositions are deprecated in that PR.

**Comment by Elijahrane** [comment]
Yeah it's probably best if this is merged after that, especially because of the current map pool I would assign chemists as:

Knight Ship: No chemist
Saltern: 1 chemist
Packed: 2 chemists

**Comment by mirrorcult** [comment]
mapping part can be done now, crash fix was pushed @Elijahrane

**Comment by Elijahrane** [comment]
> mapping part can be done now, crash fix was pushed @Elijahrane

Thanks, everything is ready now.

File: Resources/Maps/saltern.yml [hunk 1]
```diff
@@ -72527,4 +72872,87 @@ entities:
...
+  - parent: 5336
+    type: Transform
+  - canCollide: False
+    type: Physics
+- uid: 7115
+  type: SpawnPointChemist
+  components:
+  - pos: 14.5,-0.5
+    parent: 853
+    type: Transform
```
Comment (mirrorcult) [inline Resources/Maps/saltern.yml:72957]: hard to tell but this makes it seem like mapping was fucked up and the map wasn't properly paused?

**Review by mirrorcult — CHANGES_REQUESTED** [review]

File: Resources/Maps/saltern.yml (hunk 1 above)
Comment (moonheart08) [reply Resources/Maps/saltern.yml:72957]: Yea that isn't right, this looks fucked. Good catch.

Comment (Elijahrane) [reply Resources/Maps/saltern.yml:72957]: Hmm, that's strange because I never unpaused and followed the instructions on the mapping documentation exactly.

Comment (Elijahrane) [reply Resources/Maps/saltern.yml:72957]: https://github.com/space-wizards/space-station-14/pull/5501

Comment (ElectroJr) [reply Resources/Maps/saltern.yml:72957]: This was my fault. Was spawning starting entities into item slots on component startup instead of map init

**Comment by Elijahrane** [comment]
@mirrorcult I have updated the maps with the extra entities from fire cabinets removed as you requested.

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**